### PR TITLE
Update Server_Side_TLS.mediawiki

### PR DESCRIPTION
**Pull request by warburtr0n**
Added F5 BIG-IP section

### CODE REVIEW

**Comment by warburtr0n** [comment]
Hi guys - hope I've edited the doc and submitted the pull request correctly. Bit of a newb on Github. Please let me know if I've done anything wrong.

**Comment by jvehent** [comment]
Good stuff! It just needs a couple minor edits and its ready to merge :)

**Comment by warburtr0n** [comment]
Updated this branch with recommended changes, hopefully it's what you were after. I've pulled the OCSP section for this. Assuming this is committed okay I'll work on the command line stuff to create the OSCP stapling profiles and then create a new branch to add it.

File: Server_Side_TLS.mediawiki [hunk 1]
```diff
@@ -577,6 +577,106 @@ The Go standard library supports TLS1.2 and a limited subset of ECDHE and GCM ci
...
+
+iRules are F5's flexible scripting language and can be used to easily enable HSTS for any TLS website. The standard HTTP should have redirection configured to send users to the HTTPS site. The following iRule is then applied to the HTTPS virtual server to insert the HSTS header. In this example we dymanically adjust the expiry date of the HSTS header to match that of the TLS certificate minus 1 day:
+
+<source>
+when RULE_INIT {
+   set static::expires [clock scan 20160131]
+}
+
+when HTTP_RESPONSE {
+   HTTP::header insert Strict-Transport-Security "max-age=[expr {$static::expires - [clock seconds]}]; includeSubDomains"
```
Comment (jvehent) [inline Server_Side_TLS.mediawiki:676]: While it's extremely cool that iRules allow to do something like this, I think we need to stick to a single value in recommended HSTS headers. Variable values would just confuse admins that may be debugging HSTS issues.
The value currently recommended in other configuration samples is `max-age=15768000`. Could you please change your example to that?

Comment (warburtr0n) [inline Server_Side_TLS.mediawiki:676]: Good point, will get it changed ASAP...

Would it be okay to link to a page which gives a more complex example?

**Pull request closed without merging**